package.json

```diff
@@ -3,30 +3,30 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "tailwind": "npx tailwindcss -i ./styles/input.css -o ./dist/output.css --watch",
     "dev": "next dev",
-    "start": "run-p tailwind dev",
```

Comment:
No need to run `tailwindcss` command anymore. It is handled by `postcss.config.mjs`.